web/Makefile

```diff
@@ -46,11 +46,10 @@ build-kotsadm: deps
 		--config webpack.config.js \
 		--env enterprise
 
-run-unit-tests:
+.PHONY: test-unit
+test-unit:
 	yarn test:unit
 
-run-prettier-eslint:
-	yarn format
-
-run-typecheck:
-	yarn test:typecheck
\ No newline at end of file
+.PHONY: lint
+lint:
+	yarn prettier && yarn lint && yarn test:typecheck
```

Comment:
there is a consolidated command that uses `concurrently` but I think it's swallowing errors. might just get rid of the consolidated commands in package.json and advise people to use the make commands.